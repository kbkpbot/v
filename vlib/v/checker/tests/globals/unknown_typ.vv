__global global_x foo
__global (
	global_y = float(5.0)
)
