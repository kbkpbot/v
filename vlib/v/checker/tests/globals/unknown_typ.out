vlib/v/checker/tests/globals/unknown_typ.vv:1:19: error: unknown type `foo`
    1 | __global global_x foo
      |                   ~~~
    2 | __global (
    3 |     global_y = float(5.0)
vlib/v/checker/tests/globals/unknown_typ.vv:3:13: error: unknown function: float
    1 | __global global_x foo
    2 | __global (
    3 |     global_y = float(5.0)
      |                ~~~~~~~~~~
    4 | )
